bmq-sdk/src/test/java/com/bloomberg/bmq/it/util/BmqBrokerTestServer.java

```diff
@@ -214,11 +214,15 @@ public void stop() {
             if (process != null) {
                 logger.info("Stopping broker, pid [{}].", pid);
                 if (pid > 0) {
-                    Runtime.getRuntime().exec("kill -SIGINT " + pid);
+                    Runtime.getRuntime().exec("kill -SIGTERM " + pid);
                 } else {
+                    process.destroy();
```

Comment:
`destroyForcibly()` causes local storage corruption sometimes for ITs, `destroy()` might be more graceful, but it's implementation dependant according to docs